packages/lib-classifier/src/plugins/drawingTools/components/draggable/draggable.js

```diff
@@ -93,7 +93,7 @@ function draggable (WrappedComponent) {
         >
           <WrappedComponent
             ref={this.wrappedComponent}
-            dragging={dragging}
+            dragging={dragging.toString()}
```

Comment:
I'm fairly sure this is a breaking change. `dragging` is a boolean value and `'false'` evaluates to `true` in JavaScript.

Reply:
Also: very bad of past me to have missed a test that would catch this.

Reply:
I think this was meant to get rid of the React warning in the console, so what it should be is: `(dragging) ? 'true' : ''` or something  like that. Depends on how it's being used. For now, probably best to remove from this PR as it is out of scope.

Reply:
It could be that the attribute was there for debugging, so that I could see when a component was being dragged. Definitely out of scope here.

Reply:
Yes intended to get rid of console warning, apologies as definitely out of scope, will remove.

Reply:
I’ll take a look at that tomorrow. I’m pretty sure it’s left over from when I was debugging the drawing tools.